src/util/Sawyer/Attribute.C

```diff
@@ -18,6 +18,7 @@ typedef Sawyer::Container::BiMap<Id, std::string> DefinedAttributes;
 static DefinedAttributes definedAttributes;
 static Id nextId = 0;
 
+ROSE_UTIL_API
```

Comment:
Need to make sure SAWYER_EXPORT matches the value of ROSE_UTIL_API as this is being built as part of the rose util library.